frontend/src/layout/navigation-3000/sidebars/dashboardsSidebarLogic.ts

```diff
@@ -16,12 +24,30 @@ export const dashboardsSidebarLogic = kea<dashboardsSidebarLogicType>([
             ['activeScene', 'sceneParams'],
         ],
     }),
+    actions({
+        setIsSearchShown: (isSearchShown: boolean) => ({ isSearchShown }),
+        setSearchTerm: (searchTerm: string) => ({ searchTerm }),
```

Comment:
Could the actions and values related to the search be extracted to some kind of "sidebarSearchLogic"? Don't like that we have to implement the same thing separately for each sidebar logic.

Reply:
Yeah, I initially wanted to keep the values separate per dashboard… but that doesn't seem _really_ useful, and introduces a lot of complexity. Put it into `navigation3000Logic`